spinoso-string/src/lib.rs

```diff
@@ -1635,11 +1635,13 @@ impl String {
     /// ```
     /// use spinoso_string::String;
     ///
-    /// let s = String::from("hello");
-    /// assert_eq!(s.index("e", None), Some(1));
-    /// assert_eq!(s.index("lo", None), Some(3));
-    /// assert_eq!(s.index("a", None), None);
-    /// assert_eq!(s.index("l", Some(3)), Some(3));
+    /// let s = String::from("via 💎 v3.2.0");
```

Comment:
Can we use the `String::utf8` constructor here for clarity?

Reply:
Sure, fixed in 44a6a075737f9c176f3c846ac642eb87de1b9f76